src/components/graph-view.js

```diff
@@ -672,6 +659,16 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       onBackgroundClick(xycoords[0], xycoords[1], d3.event);
     }
 
+    if (
+      !d3.event.shiftKey &&
+      d3.event.type === 'contextmenu' &&
+      onContextMenu
+    ) {
+      const xycoords = d3.mouse(d3.event.target);
+
+      onContextMenu(xycoords[0], xycoords[1], d3.event);
```

Comment:
did we check with the graph controls slider, or we dont have plan to support that. Also do we not need that fit to screen option

Reply:
We don't need to show the graph controls slider and the zoom to fit option will be part of the context menu.